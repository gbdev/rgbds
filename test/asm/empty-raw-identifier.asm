MACRO #macro
	println "all args: \#"
	println "bad args: \<?>, \<#>"
ENDM
	#macro a, #b, c, 1, #2, 3
