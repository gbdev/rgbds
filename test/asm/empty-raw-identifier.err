error: empty-raw-identifier.asm(5) -> empty-raw-identifier.asm::macro(3):
    Invalid character in bracketed macro argument '?'
error: empty-raw-identifier.asm(5) -> empty-raw-identifier.asm::macro(3):
    Empty raw symbol in bracketed macro argument
error: Assembly aborted (2 errors)!
