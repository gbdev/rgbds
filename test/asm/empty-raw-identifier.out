all args: a,#b,c,1,#2,3
bad args: >, >
